### ISSUE TITLE
Question: any way to remove system apps using package name, directly?

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [Magisk-Modules-Repo/terminal_debloater_magisk#38]
For example, I wanted to remove "YouTube Music", so I typed:

debloat -d com.google.android.apps.youtube.music

But it failed:

![image](https://user-images.githubusercontent.com/5357526/86520223-b0208500-be4a-11ea-9e65-87bf53945d9c.png)

Using a package name is more correct than a name, because a name might change, and a name might exist for other apps too.
Using a package name, I could also create a batch file to remove multiple apps.

**Comment by b3wii** [comment]
Yes and in addition it would be nice to somewhere see the real package names:

```
17 - Files (HwCloudDrive)
18 - Files (HwCloudDrive)
30 - Huawei Share (HuaweiShare)
31 - Huawei Share (HuaweiShare)
```
